textproc/pandoc/Portfile

```diff
@@ -24,33 +25,45 @@ long_description    \
     Textile, groff man pages, plain text, Emacs Org-Mode, AsciiDoc, EPUB (v2 \
     and v3), FictionBook2, and S5, Slidy and Slideous HTML slide shows.
 
-depends_lib-append  \
-                    port:hs-aeson \
-                    port:hs-alex \
-                    port:hs-attoparsec \
-                    port:hs-base64-bytestring \
-                    port:hs-blaze-html \
-                    port:hs-blaze-markup \
-                    port:hs-data-default \
-                    port:hs-extensible-exceptions \
-                    port:hs-happy \
-                    port:hs-highlighting-kate \
-                    port:hs-hslua \
-                    port:hs-http \
-                    port:hs-mtl \
-                    port:hs-network \
-                    port:hs-pandoc-types \
-                    port:hs-parsec \
-                    port:hs-random \
-                    port:hs-syb \
-                    port:hs-sha \
-                    port:hs-tagsoup \
-                    port:hs-temporary \
-                    port:hs-texmath \
-                    port:hs-text \
-                    port:hs-unordered-containers \
-                    port:hs-vector \
-                    port:hs-xml \
-                    port:hs-yaml \
-                    port:hs-zip-archive \
-                    port:hs-zlib
+depends_build-append  \
+                    port:stack
+
+set system_gcc      /usr/bin/gcc
+
+use_configure       no
+
+# the PATH environment must provide /usr/bin/gcc, not ${prefix}/bin/gcc
+build.env \
+    PATH=/usr/bin:$env(PATH) \
+    PREFIX=${prefix} \
+    CC=${system_gcc}
+
+build {
+    xinstall -m 0755 -d ${workpath}/.stack
+    # the PATH environment must use /usr/bin/gcc, not ${prefix}/bin/gcc
+    system -W ${worksrcpath} \
+        "${prefix}/bin/stack setup --stack-root ${workpath}/.stack \
+                --with-gcc ${system_gcc} --allow-different-user \
+            && ${prefix}/bin/stack build --stack-root ${workpath}/.stack \
+                --with-gcc ${system_gcc} --allow-different-user \
+            && ${prefix}/bin/stack install --stack-root ${workpath}/.stack \
+                --local-bin-path ${worksrcpath}/bin \
+                --with-gcc ${system_gcc} --allow-different-user"
+}
+
+destroot {
+    xinstall -m 755 -W ${worksrcpath}/bin \
+        ${name} \
+        ${destroot}${prefix}/bin
+}
+
+# the PATH environment must provide /usr/bin/gcc, not ${prefix}/bin/gcc
+test.env \
+    PATH=/usr/bin:$env(PATH) \
+    PREFIX=${prefix} \
+    CC=${system_gcc}
+
+test {
```

Comment:
I guess you need `test.run yes` to enable tests?

Reply:
Okay, but what am I supposed to say? I though that writing the `test` block did this, but no?

Reply:
All the above does is define what running a test is supposed to do. It does not activate running it by default. For that just add

```
test.run yes
```

to the port file